constitution.tex

```diff
@@ -479,6 +482,35 @@ \subsection{Acceptance of Amendments} \label{sect:amending:acceptance}
     \item Before the proposed amendment shall become operational, it must be approved by the Club Advisor.
 \end{enumerate}
 
+% Article: adherence to University Policy
+\section{Adherence to University Policy} \label{art:university-policy}
+
+\subsection{Anti-Hazing} \label{sect:university-policy:anti-hazing}
+\underline{\textbf{Per the RIT Hazing Policy}} (\textit{RIT Student Conduct Process; IV. RIT Code of Conduct; 14. Hazing/Failure to Report Hazing}) \\
+
+\noindent Hazing/Failure to Report Hazing.
+Behavior, regardless of intent, which endangers the emotional, or physical health and safety of a Student for the purpose of membership, affiliation with, or maintaining membership in, a group or Student Organization.
+Hazing includes any level of participation, such as being in the presence, having awareness of hazing, or failing to report hazing.
+Examples of hazing include, but are not limited to, beating or branding, sleep deprivation or causing excessive fatigue, threats of harm, forcing or coercing consumption of food, water, alcohol or other drugs, or other substances, verbal abuse, embarrassing, humiliating, or degrading acts, or activities that induce, cause or require the Student to perform a duty or task which is not consistent with fraternal law, ritual or policy or involves a violation of local, state or federal laws, or the RIT Code of Conduct. \\
+
+\noindent \underline{\textbf{NY State Hazing Law}} \\
+\noindent \textsection 120.16 Hazing in the first degree.
+A person is guilty of hazing in the first degree when, in the course of another person's initiation into or affiliation with any organization, he intentionally or recklessly engages in conduct which creates a substantial risk of physical injury to such other person or a third person and thereby causes such injury.
+Hazing in the first degree is a class A misdemeanor. \\
+
+\noindent \textsection 120.17 Hazing in the second degree.
+A person is guilty of hazing in the second degree when, in the course of another person's initiation or affiliation with any organization, he intentionally or recklessly engages in conduct which creates a substantial risk of physical injury to such other person or a third person.
+Hazing in the second degree is a violation. 
+
+\subsection{Anti-Discrimination Clause} \label{sect:university-policy:anti-discrimination}
+
+This organization shall not discriminate on the basis of sex, race, color, sexual orientation gender identity and gender expression, religion, age marital sate, national origin, disability or veteran status.
```

Comment:
Should have an oxford comma

Reply:
Both these grammar issues are from the CCL documents that said to copy verbatim, so...

Fixed them anyways, should be all set.